### PR TITLE
New version: Synology.ChatClient version 1.2.0-146

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
﻿- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26421)

### CODE REVIEW

File: manifests/s/Synology/ChatClient/1.2.0-146/Synology.ChatClient.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PackageIdentifier: Synology.ChatClient
+PackageVersion: 1.2.0-146
+PackageLocale: en-US
+Publisher: Synology
+PublisherUrl: https://www.synology.com
+PublisherSupportUrl: https://www.synology.com/company/contact_us
+PrivacyUrl: https://www.synology.com/privacy
+PackageName: Synology Chat
+PackageUrl: https://kb.synology.com/en-global/DSM/help/ChatClient/chatclient
+License: Copyright (c) Synology Inc.
```
Comment (quhxl) [inline manifests/s/Synology/ChatClient/1.2.0-146/Synology.ChatClient.locale.en-US.yaml:13]:
```suggestion
License: Proprietary
LicenseUrl: https://www.synology.com/en-global/company/legal/terms_EULA
Copyright: Copyright (c) Synology Inc.
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/26421#discussion_r699431178

**Comment by vedantmgoyal2009** [comment]
As I don't know much about licensing, can you tell what's the difference between the two.
```
License: Copyright (c) Synology Inc.
License: Proprietary
```

**Comment by quhxl** [comment]
This is the discussion about this: #12254

**Review by quhxl — APPROVED** [review]

**Comment by quhxl** [comment]
grafik

**Comment by OfficialEsco** [comment]
Copyright is not a License, its a License Type.
So if we can, we prefer to not use that in the License field.
For example for future filtering options like [Alternativeto.net](https://alternativeto.net/)

https://en.wikipedia.org/wiki/Proprietary_software (You can also see the different License Types here)
> Proprietary software, also known as non-free software or closed-source software, is computer software for which the software's publisher or another person reserves some rights from licenses to use, modify, share modifications, or share the software. It is the opposite of Open Source. Sometimes includes patent rights.

**Comment by vedantmgoyal2009** [comment]
> > Proprietary software, also known as non-free software or closed-source software, is computer software for which the software's publisher or another person reserves some rights from licenses to use, modify, share modifications, or share the software. It is the opposite of Open Source. Sometimes includes patent rights.

According to this, most (majority of the software in the repo) are Proprietary.

**Comment by OfficialEsco** [comment]
> According to this, the majority of the software in the repo is Proprietary.

I'm not 100% sure, yes A LOT of big Softwares are Proprietary, but we do have around 900 GitHub Releases and 110 SourceForge releases in the repo

**Pull request merged**